src/layouts/Footer.js

```diff
@@ -2,7 +2,8 @@ import { makeStyles } from "@material-ui/core/styles";
 import Link from "@material-ui/core/Link";
 import Grid from "@material-ui/core/Grid";
 import moment from "moment";
-import { Link as RouterLink } from "react-router-dom";
+import { Link as RouterLink, Route } from "react-router-dom";
+import { routes } from "../routes/routing";
```

Comment:
You have 2 files routing.js now, maybe just folder routing and inside routes with, well, routes :) and remove empty file from other folder

Reply:
Removed route import, changed folder to routing and put routes.js there.